packages/apollo-link-ws/README.md

```diff
@@ -20,10 +20,13 @@ const client = new SubscriptionClient(GRAPHQL_ENDPOINT, {
   reconnect: true,
 });
 
-const link = new WebSocketLink(client);
+const link = new WebSocketLink(paramsOrClient, options);

```

Comment:
we should make the `options` argument optional so it isn't a breaking change

Reply:
done that too options are now optional